src/Microsoft.Framework.PackageManager/Utils/LockFileUtils.cs

```diff
@@ -63,9 +65,7 @@ public static LockFileTargetLibrary CreateLockFileTargetLibrary(IPackage package
             // it has the correct casing that runtime needs during dependency resolution.
             lockFileLib.Name = correctedPackageName ?? package.Id;
             lockFileLib.Version = package.Version;
-
-            var files = package.GetFiles().Select(p => p.Path.Replace(Path.DirectorySeparatorChar, '/')).ToList();
-
+            var files = library.Files.Select(p => p.Replace(Path.DirectorySeparatorChar, '/'));
```

Comment:
Could we do this up front (in line 35)? `Path.Combine` should work with well with forward slashes.